runtime/gc_modron_startup/mminit.cpp

```diff
@@ -3336,6 +3342,12 @@ setIndexableObjectHeaderSizeWithoutDataAddress(J9JavaVM* vm)
 		vm->contiguousIndexableHeaderSize = sizeof(J9IndexableObjectContiguousFull);
 		vm->discontiguousIndexableHeaderSize = sizeof(J9IndexableObjectDiscontiguousFull);
 	}
+
+	if (MM_GCExtensions::getExtensions(vm)->isVirtualLargeObjectHeapEnabled) {
+		vm->unsafeIndexableHeaderSize = 0;
+	} else {
+		vm->unsafeIndexableHeaderSize = vm->contiguousIndexableHeaderSize;
+	}
```

Comment:
you can move this new block to the caller and have it only once, since it's identical for with/without DataAddr cases

Reply:
actually, this is not needed for without dataAddr case - we can't have offheap without dataAddr field
assuming that unsafeIndexableHeaderSize (like any javavm field) defaults to 0